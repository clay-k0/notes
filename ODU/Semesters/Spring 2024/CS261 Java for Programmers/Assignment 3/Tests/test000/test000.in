3
Marksman 10
EagleEye 10 Marksman
Sniper 90 EagleEye
Agile 30
Fleetfooted 75 Agile Marksman
Woodsman 30 Agile Fleetfooted

