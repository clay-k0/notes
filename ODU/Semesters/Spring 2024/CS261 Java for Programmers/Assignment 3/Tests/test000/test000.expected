115:  Agile:30  Fleetfooted:75  Marksman:10
